akka-docs/rst/scala/code/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala

```diff
@@ -95,4 +96,36 @@ class DebuggingDirectivesExamplesSpec extends RoutingSpec {
       responseAs[String] shouldEqual "logged"
     }
   }
+  "logRequestResultWithResponseTime" in {
+
+    def akkaResponseTimeLoggingFunction(loggingAdapter: LoggingAdapter,
+                                        requestTimestamp: Long,
+                                        level: LogLevel = Logging.InfoLevel)(req: HttpRequest)(res: Any): Unit = {
+      val entry = res match {
+        case Complete(resp) =>
+          val responseTimestamp: Long = System.nanoTime
+          val elapsedTime: Long = (responseTimestamp - requestTimestamp) / 1000
+          val loggingString = s"""Logged Request:${req.method}:${req.uri}:${resp.status}:${elapsedTime}"""
+          LogEntry(loggingString, level)
+        case anythingElse =>
+          LogEntry(s"$anythingElse", level)
```

Comment:
It isn't really anything else, but a `Rejected(reasons)`. If it can happen depends on where it is put in the routes. So maybe capture that with the case clause just for clarity?